src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -1622,31 +1622,49 @@ def test_subscription_level_what_if_ts(self, resource_group, resource_group_loca
 
 
 class DeploymentScriptsTest(ScenarioTest):
-    @ResourceGroupPreparer(name_prefix='cli_test_deployment_scripts')
+    @ResourceGroupPreparer(name_prefix='cli_test_deployment_scripts', location='brazilsouth')
```

Comment:
There's a known issue in ARM that "list requests" when the RP has more than one endpoint it ends up duplicating the resources, brazil south doesn't have multiple endpoints therefore used this region to ensure a list request only returns one resource (in case you wonder why we are using a specific location).